### DISCUSSION TITLE
Export CSV in Grafana as per Time correction settings

### DISCUSSION THREAD

**Discussion by ankaggar** [grafana/grafana#36969] [General]
Hi Team,
Whenever I am Exporting CSV for any graph, the time it shows is GMT time zone, while when I can see in the graph, it shows browser's time zone because I have selected Browser in Time Correction Setting.
Can't we use the same logic for downloading CSV as well?

Any help would be appreciated.
- What grafana version are you using?
  Grafana version: 1.9.0  
- What datasource are you using?
  NA
- What OS are you running grafana on?
  Linux
- What did you do?
  Chose the Time Correction as Browser
- What was the expected result?
  CSV file should be generated in Browser timing or as per Time correction setting.
- What happened instead?
  Time is coming in CSV file in UTC format instead of Browser time, although the Time correction setting is Browser.

**Comment by sebbalex** [comment]
any update on this?

**Comment by chaitanya-ather** [comment]
+1 for this.
Downloaded csv data is almost misleading because we are expecting the data appear with the same time zone as viewed in the browser.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
